src/plugins/coincapLegacy.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import {
+  type EdgeCorePluginOptions,
+  type EdgeRatePlugin
+} from 'edge-core-js/types'
+
+export function makeCoincapLegacyPlugin (
+  opts: EdgeCorePluginOptions
+): EdgeRatePlugin {
+  const { io } = opts
+
+  return {
+    rateInfo: {
+      displayName: 'CoincapLegacy'
+    },
+
+    async fetchRates (pairsHint) {
+      const reply = await io.fetch('https://coincap.io/front')
+      const json = await reply.json()
+
+      // Grab all the pairs which are in USD:
+      const pairs = []
+      for (const entry of json) {
+        if (typeof entry.short !== 'string') continue
+        if (typeof entry.price !== 'number') continue
+        const currency = entry.short
+
+        pairs.push({
```

Comment:
The idea was that we would filter the list of currencies to just the missing ones. Otherwise, we are back to the original problem of bad rate data for things like BTC. Please coordinate with Matthew to get a list of missing coins, and only push those to the output list.